ZLPeoplePickerViewController/Internal/ZLBaseTableViewController.m

```diff
@@ -156,6 +156,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView
         cell.accessoryType = UITableViewCellAccessoryNone;
         // cell.selectionStyle = UITableViewCellSelectionStyleDefault;
     }
+    
+    [cell setSelectionStyle:UITableViewCellSelectionStyleNone];
```

Comment:
Thank you! Your changes look great but I have a question about this line: 

The address book in the phone app uses the default value. What is the intention for changing it to `None`?